package.json

```diff
@@ -1,6 +1,6 @@
 {
   "name": "ffc-pay-statement-constructor",
-  "version": "0.3.2",
+  "version": "0.3.4",
```

Comment:
I'd argue this should be `0.4.0` as it's a new feature added as opposed to patching what's already there